mesonbuild/mconf.py

```diff
@@ -205,7 +205,10 @@ def print_options(self, title: str, options: 'coredata.KeyedOptionDictType') ->
                 printable_value = '<inherited from main project>'
             self.add_option(str(root), o.description, printable_value, o.choices)
 
-    def print_conf(self):
+    def print_conf(self, no_pager: bool):
+        if not no_pager:
```

Comment:
This feels awkward. I think we want a `pager: bool` argument, which also implies that argparse should parse --no-pager into options.pager (default to True, store_false).

Reply:
Makes sense, done.